app/models/form_profiles/va_526ez.rb

```diff
@@ -281,7 +281,7 @@ def prefill_military_address(address)
   end
 
   def initialize_payment_information
-    return {} unless user.authorize(:ppiu, :access?) && user.authorize(:evss, :access?)
+    return {} unless user.authorize(:lighthouse, :direct_deposit_access?) && user.authorize(:evss, :access?)
```

Comment:
Here's the LH policy: 
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/policies/lighthouse_policy.rb.
And you can compare it to the PPIU policy:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/policies/ppiu_policy.rb